diff --git a/router/router.go b/router/router.go
--- a/router/router.go
+++ b/router/router.go
@@ -11,5 +11,6 @@ import (
 	"github.com/pterodactyl/wings/config"
 	"github.com/pterodactyl/wings/remote"
 	"github.com/pterodactyl/wings/router/middleware"
+	ibyterestic "github.com/pterodactyl/wings/ibyte/restic"
 	wserver "github.com/pterodactyl/wings/server"
 )
@@ -110,9 +111,11 @@ func Configure(m *wserver.Manager, client remote.Client) *gin.Engine {
 		backup := server.Group("/backup")
 		{
 			backup.POST("", postServerBackup)
 			backup.POST("/:backup/restore", postServerRestoreBackup)
 			backup.DELETE("/:backup", deleteServerBackup)
 		}
+
+		ibyterestic.Register(server.Group("/restic"))
 	}
 
 	return router
